features/layer_lock.c

```diff
@@ -84,12 +118,11 @@ void layer_lock_invert(uint8_t layer) {
 
 // Implement layer_lock_on/off by deferring to layer_lock_invert.
 void layer_lock_on(uint8_t layer) {
-  if (!is_layer_locked(layer)) { layer_lock_invert(layer); }
+  if (!is_layer_locked(layer)) {layer_lock_invert(layer); }
```

Comment:
Formatting nit: Surely this is unintended, the diff shows a space got removed just after the `{`. For simpler diff, please change it back to

```c
if (!is_layer_locked(layer)) { layer_lock_invert(layer); }
```

Reply:
My mistake. I had added something inside the layer_lock_on feature at one point and I must not have restored it to pre-edit state.